Cargo.toml

```diff
@@ -35,6 +35,10 @@ spin = { version = "*", default-features = false }
 sha2 = {version = "0.10", default-features = false }
 test_workloads = { path = "./test-workloads" }
 
+[features]
+default=[]
+vectors = []
```

Comment:
Instead of defining a feature for vectors and using that in all the `#[cfg]` directives, could you just use the target?